webpack.config.js

```diff
@@ -21,20 +40,34 @@ module.exports = (env) => {
               }
             },
             {
-              loader: path.resolve(__dirname, 'loader.js'),
+              loader: path.resolve(__dirname, 'webpack_utils/reactRenderLoader.js'),
               options: {
                 env,
               }
             },
             {
               loader: 'babel-loader',
               options: {
-                presets: ['@babel/preset-env']
+                presets: ['@babel/preset-env', '@babel/preset-react'],
               }
             }
           ]
         }
       ]
-    }
+    },
+    plugins: [
+      new InlineCssPlugin(),
+      // the htmlWebpackPlugin requires one instance for each template
+      ...templates.map((templateName) => (
+        new HtmlWebpackPlugin({
+          filename: path.resolve(__dirname, 'dist', `final_template_${templateName}.html`),
+          inject: false,
+          templateContent: template,
```

Comment:
does this force us to have the same structure as `webpack_utils/template.js`?

Reply:
You mean in each template? Nope.